Unused variables are reported twice

When a TypeScript file has an unused variable in it, the error is reported twice: once for the standard `no-unused-vars` rule and again for the `@typescript-eslint/no-unused-vars` rule.

```
  1:8  error  '_' is defined but never used  no-unused-vars
  1:8  error  '_' is defined but never used  @typescript-eslint/no-unused-vars
```

We should determine whether or not we can disable `no-unused-vars` for TypeScript files.

--- a/index.js
+++ b/index.js
@@ -262,6 +262,7 @@ module.exports = {
             'no-undef': 'off',
             // The standard ESLint `no-unused-vars' rule will throw false positives with
             // class properties in TypeScript. The TypeScript-specific rule fixes this.
+            'no-unused-vars': 'off',
             '@typescript-eslint/no-unused-vars': 'error',
             // new-cap throws errors with property decorators
             'new-cap': 'off',

